tests/providers/openai/operators/test_openai.py

```diff
@@ -34,3 +36,30 @@ def test_execute_with_input_text():
     embeddings = operator.execute(context)
 
     assert embeddings == [1.0, 2.0, 3.0]
+
+
+def test_execute_with_invalid_input_empty_string():
```

Comment:
Maybe we can combine these 3 tests into a single parameterized test, WDYT?

Reply:
Something like this - https://github.com/apache/airflow/blob/4cc98ba6648b4a350f9093c4e22ad6cff87ac5e4/tests/providers/amazon/aws/sensors/test_ecs.py#L198

Reply:
Definitely! great recommendation!

Reply:
Done